.github/workflows/chromatic_pr_on_push.yml

```
@@ -12,10 +12,14 @@ jobs:
           fetch-depth: 0
       - name: Setup
         uses: ./.github/actions/setup
+      - name: Execute Convex
+        run: pnpm convex:dev
       - name: Storybook Build  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)